src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class Main {

```

Comment:
It is not necessary to create this class, for testing there is already a created class by path:
/src/test/java/core/basesyntax/WorkWithFileTest.java